firebase-firestore/src/main/java/com/google/firebase/firestore/local/LocalStore.java

```diff
@@ -282,6 +282,14 @@ public LocalWriteResult writeLocally(List<Mutation> mutations) {
         });
   }
 
+  /**
+   * Returns the largest (latest) batch id in mutation queue that is pending server response.
+   * Returns 0 if the queue is empty.
```

Comment:
Can we return -1 to clearly differentiate an empty queue and the case where we are writing for the first write? Returning `0` requires some internal knowledge about how we assign batch IDs. What's even better is that we have a constant for this: https://github.com/firebase/firebase-android-sdk/blob/69f29d43f4b4e14bed7f5c44a1dc3883c11cce57/firebase-firestore/src/main/java/com/google/firebase/firestore/model/mutation/MutationBatch.java#L41

Reply:
When would you need this information, when is checking if it's -1 or 0 helpful?

If we don't know a case when this will be useful, i'd like to keep the abstraction simple.